### DISCUSSION TITLE
Home and End keys don't work in search box

### DISCUSSION THREAD

**Discussion by uta-danevil** [mdn/mdn-community#318] [Platform]
Hi, i just noticed in the MDN search box, the home key and the end key doesn't work. 

Example: Type something in the search box. Now press `<home>`. It should go to the start of the search box. But it doesn't. Same thing with `<end>`

I often use this feature to select text in a textbox (for example pressing `<shift>+<home>` should select from the current position to the start of the textbox).

**Comment by manjuu-eater** [comment]
It is because Home key is used for moving selection in popup.
![searching image](https://user-images.githubusercontent.com/88940383/213623116-7faba18b-02ec-4355-a3d6-bc7becb34b9e.png)

I think `<shift>+<home>` must not sent to popup move.
